api/v1alpha1/common.go

```diff
@@ -221,6 +230,34 @@ func filterMatchVersionConstraint(cond FilterCond, filter Filter) bool {
 	return false
 }
 
+func Match(fc map[string]FilterCond, filter Filter) bool {
```

Comment:
IMO, need handle FilterOpKeep and FilterOpIgnore

Reply:
https://github.com/kubebb/core/pull/41/files#diff-2e9abd09c0abdabe2a30a80ced0000f009dd3e909fc18ad52b113e1189e6affbR303 The ignore case is finished here, so only the Keep case is matched in Match.

Reply:
IMO, if we add a `FilterOp` field into `FilterCond` struct which is a arg to func `Match`, we should handle it in `Match` function, so that we can just use `Match` function.